build/nuget-release.yml

```diff
@@ -0,0 +1,163 @@
+name: $(Package.Version)
+
+trigger: none
+
```

Comment:
What is the default for PR? Maybe good to explicitly add it here with `none`?

Reply:
:+1: